src/app/core/services/api/fake-backend/transfer-history-fake-service.ts

```diff
@@ -67,8 +66,8 @@ export let transferHistoryEntries: TransferHistoryEntry[] = [
     contractAgreementId: 'test-asset-4-cd:f52a5d30-6356-4a55-a75a-3c45d7a88c3e',
     direction: 'PROVIDING',
     counterPartyConnectorEndpoint: 'https://sovity-demo2-edc/api/v1/ids/data',
-    assetName: TestAssets.full.properties[AssetProperties.name],
-    assetId: TestAssets.full.properties[AssetProperties.id],
+    assetName: TestAssets.full.name || TestAssets.full.assetId,
```

Comment:
redundant expression, can be just TestAssets.full.name (as we know its contents)